### ISSUE TITLE
CI: failure on concurrent usage of frontend-maven-plugin to `yarn install`

### ISSUE THREAD

**Issue by cmur2** [camunda/camunda#26327]
## Description

Follow-up to #26288

[Houssain discovered](https://camunda.slack.com/archives/C071KP5BTHB/p1734683891745539) that the problem spotted and mitigated in #26288 could actually result from Maven concurrently running for `operate-webjar` project and (Optimize) `client` project. This can be seen in https://github.com/camunda/camunda/actions/runs/12351334347/job/34466665929 where those are the only failing projects:

```
Error:  Failed to execute goal com.github.eirslett:frontend-maven-plugin:1.15.1:yarn (yarn install) on project operate-webjar: Failed to run task: 'yarn ' failed. org.apache.commons.exec.ExecuteException: Process exited with an error: 1 (Exit value: 1) -> [Help 1]
...
Error:  Failed to execute goal com.github.eirslett:frontend-maven-plugin:1.15.1:yarn (yarn install) on project client: Failed to run task: 'yarn ' failed. org.apache.commons.exec.ExecuteException: Process exited with an error: 1 (Exit value: 1) -> [Help 1]
...
```

This wasn't a problem in the `operate-webjar` project while there was no other project using it at the same time. This changed when Optimize became part of the single JAR build process in #25204 and lead to the failure.

## Goal

* check if `operate-webjar` project is still needed given that https://github.com/camunda/camunda/issues/17926 was implemented
* migrate away from using frontend-maven-plugin to `yarn install` which is prone to concurrency issues (disabling concurrency in a monorepo is not sustainable)

## Hints

The CI changes related to this ticket should be backported to all applicable `stable/*` branches.

**Comment by maxdanilov** [comment]
since Optimize is not a part of the single JAR anymore - is this ticket still relevant?

**Comment by cmur2** [comment]
> since Optimize is not a part of the single JAR anymore - is this ticket still relevant?

Agreed, also the fix in https://github.com/camunda/camunda/pull/26409 by Meggle probably resolved that since I personally didn't notice any recent builds with that error.
